docs/extensions/add-operator/index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Add Operators
```

Comment:
This can be one article about how to add a custom op, no need for 3

Reply:
sounds good, updated!